crypto/conf/conf_api.c

```diff
@@ -63,6 +63,7 @@
 # define NDEBUG
 #endif
 
+#include "cryptlib.h"
```

Comment:
most files include all system headers (stdlib.h, string.h etc.) before "cryptlib.h"

Reply:
Shuffled it down...